platform/xdebugger-impl/src/com/intellij/xdebugger/impl/XDebuggerActionsCollector.kt

```diff
@@ -0,0 +1,90 @@
+// Copyright 2000-2023 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license.
+package com.intellij.xdebugger.impl
+
+import com.intellij.internal.statistic.collectors.fus.fileTypes.FileTypeUsagesCollector
+import com.intellij.internal.statistic.eventLog.EventLogGroup
+import com.intellij.internal.statistic.eventLog.events.EventFields
+import com.intellij.internal.statistic.eventLog.events.EventFields.Int
+import com.intellij.internal.statistic.eventLog.events.EventFields.IntList
+import com.intellij.internal.statistic.eventLog.events.EventFields.String
+import com.intellij.internal.statistic.eventLog.events.EventFields.StringListValidatedByCustomRule
+import com.intellij.internal.statistic.eventLog.validator.ValidationResultType.ACCEPTED
+import com.intellij.internal.statistic.eventLog.validator.rules.EventContext
+import com.intellij.internal.statistic.eventLog.validator.rules.impl.CustomValidationRule
+import com.intellij.internal.statistic.service.fus.collectors.CounterUsagesCollector
+import com.intellij.xdebugger.frame.XStackFrame
+
+private const val UNKNOWN_TYPE = "Unknown"
+
+class XDebuggerActionsCollector : CounterUsagesCollector() {
+  companion object {
+
+    private val group = EventLogGroup("xdebugger.actions", 3)
```

Comment:
This won't work this way.
You need to rename `group` to `GROUP` and implement `override fun getGroup(): EventLogGroup = GROUP` in `XDebuggerActionsCollector` class.

Reply:
Done